client/src/main/scala/com/dwolla/cloudflare/domain/model/accounts/AccountMember.scala

```diff
@@ -1,12 +1,14 @@
 package com.dwolla.cloudflare.domain.model.accounts
 
+import com.dwolla.cloudflare.BaseUrl
+
 case class AccountMember (
   id: String,
   user: User,
   status: String,
   roles: Seq[AccountRole]
 ) {
   def uri(accountId: String): String = {
-    s"https://api.cloudflare.com/client/v4/accounts/$accountId/members/$id"
+    (BaseUrl / "accounts" / accountId / "members" / id).toString()
```

Comment:
Why do you want to return a String here?

Reply:
it was already returned as a string, wasn't going to make a breaking change. I believe its used as the physical resource id. I can change it to return a URI